public/robots.public.txt

```diff
@@ -23,7 +22,6 @@ Disallow: /*?*query=
 Disallow: /*?*sort_
 Disallow: /*?*selected_tags
 Disallow: /*?*selected_pseuds
-Disallow: /*?*use_openid
```

Comment:
Would you mind also removing `Disallow: /*?*selected_pseuds` and `Disallow: /people?*show=`? I unfortunately didn't get the issue updated to mention those before you submitted your pull request, but I can update it now.

Reply:
I'd be happy too. Sorry for the fast response. I saw a way of blocking chatGPT and wanted to get this taken care of. lol